tests/TestCase/View/Helper/BreadcrumbsHelperTest.php

```diff
@@ -12,7 +13,7 @@ class BreadcrumbsHelperTest extends TestCase
     public $View;
 
     /**
-     * @var HtmlHelper
+     * @var BreadcrumbsHelper
```

Comment:
Nitpick: Needs to be FQCN to be actually useful to IDEs :slightly_smiling_face:

Reply:
And the non-PhpStorm peasants should suffer, for they have sinned :grimacing: 

I can change it, but none of the the other `@var` tags in the test files uses fully qualified names. I can change them all if you want?

Reply:
You could just use my Spryker sniffer which fixes it for you globally :P

Reply:
It's a test file so I don't really care :P